genome_kit/genome_track.py

```diff
@@ -568,6 +568,8 @@ def __call__(self, interval, dtype=None):  # pragma: no cover
             The stranded query interval.
         dtype : :py:class:`type`
             Optional. The numpy dtype of the resulting array.
+        dtype : :py:class:`~numpy.ndarray`
```

Comment:
I think there is a minor typo here! 
```suggestion
        out : :py:class:`~numpy.ndarray`
```